.github/workflows/ci_test-full.yml

```diff
@@ -136,6 +136,11 @@ jobs:
       run: |
         python requirements/check-avail-extras.py
 
+    # FIXME
+    - name: Test init_meta_context
+      run: |
+        PL_RUN_STANDALONE_TESTS=1 python -m pytest tests/utilities/test_meta.py::test_fit_with_init_meta_context -vv
```

Comment:
I gues you can run this locally as it does not require any GPU :rabbit:

Reply:
Yes, I'm debugging it locally now. I just wanted to check environments in which the bug occurs and also wanted to share the result with other people if needed.